test/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/SelectDutiesControllerSpec.scala

```diff
@@ -114,6 +115,14 @@ class SelectDutiesControllerSpec
   def getBackLink(document: Document): String =
     document.select("a.govuk-back-link").attr("href")
 
+  def performAction(): Future[Result] = controller.selectDuties()(FakeRequest())
+
+  def getHintText(document: Document, hintTextId: String) = {
+    val hintTextElement = document.select(s"div#$hintTextId")
+
+    if (hintTextElement.hasText) Some(hintTextElement.text()) else None
```

Comment:
Can be done also Option(hasTextElement).filter(_.hasText).map(_.text())

Reply:
It's a cleverer solution but harder to follow especially for people new to Scala. I'll keep it simple.

Reply:
Np